softwerkskammer/lib/mailsender/mailsenderService.js

```diff
@@ -38,6 +40,41 @@ function activityMarkdown(activity, language) {
   return markdown;
 }
 
+function createChunkedSendingReportMessage(statusmessages, subject, sender) {
+  const anySendingError = R.any(statusmessage.isErrorMessage, statusmessages);
+  const resultMessage = new Message(
+    {
+      subject,
+      markdown: anySendingError
+        ? `Fehler: ${statusmessages.map((s) => s.contents().additionalArguments.err).join(" ")}`
+        : "E-Mails erfolgreich versendet",
+    },
+    sender,
+  );
+  resultMessage.setTo(sender.email());
+  return resultMessage;
+}
+
+function sendMailInChunks(maxMailSendingChunkSize, allMembers, message, type, sender) {
+  const membersInChunks = R.splitEvery(maxMailSendingChunkSize, allMembers);
+
+  Promise.all(
+    membersInChunks.map((bccs) => {
+      message.setBccToMemberAddresses(bccs);
```

Comment:
Hier werden die Duplikate beseitigt, allerdings in der neuen Version kann derselbe member in mehreren chunks sein!

Reply:
Stimmt, nicht dran gedacht. Ist war etwas unschön, aber dann muss ich das Duplikatefiltern eben im Service machen...

Reply:
Erledigt